sympy/physics/vector/functions.py

```diff
@@ -609,5 +608,5 @@ def dynamicsymbols(names, level=0):
         return reduce(diff, [t] * level, esses(t))
 
 
-dynamicsymbols._t = Symbol('t')
+dynamicsymbols._t = symbols('t')
```

Comment:
This is not needed, is it? You can still import `Symbol` from `sympy.core.backend`, cannot you?